Accompany/Controllers/Core/Home/AddNoteDetailsTableViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  SemesterDetailsTableViewController.swift
+//  Accompany
+//
+//  Created by Kelbin David on 2022-02-10.
+//
+
+import UIKit
+import SnapKit
+
+protocol AddNoteDetailsTableViewControllerDelegate {
+  func add(todo: ToDo)
+  func edit(todo: ToDo)
+}
+
+class AddNoteDetailsTableViewController: UITableViewController {
+  
+  var saveBarButton: UIBarButtonItem = {
+    let barButton = UIBarButtonItem(barButtonSystemItem: .save, target: self, action: #selector(saveButtonTapped))
+    return barButton
+  }()
+  
+  var todo: ToDo?
+  var delegate: AddNoteDetailsTableViewControllerDelegate?
+  var isCompleteButton: UIButton?
+  var toDoTitleTextField: UITextField?
+  var noteTextView: UITextView?
+  let buttonIndexPath = IndexPath(row: 0, section: 0)
+  let noteIndexPath = IndexPath(row: 0, section: 1)
```

Comment:
Please refactor "noteIndexPath" to "notesCellIndexPath".
```suggestion
  let notesCellIndexPath = IndexPath(row: 0, section: 1)
```